packages/dev/s2-docs/src/ColorSearchView.tsx

```diff
@@ -0,0 +1,466 @@
+  'disabled': colorSwatch('disabled')
+};
+
+const textSwatches = {
+  'accent': colorSwatch('accent', 'color'),
+  'neutral': colorSwatch('neutral', 'color'),
+  'neutral-subdued': colorSwatch('neutral-subdued', 'color'),
+  'negative': colorSwatch('negative', 'color'),
+  'disabled': colorSwatch('disabled', 'color'),
+  'heading': colorSwatch('heading', 'color'),
+  'title': colorSwatch('title', 'color'),
+  'body': colorSwatch('body', 'color'),
+  'detail': colorSwatch('detail', 'color'),
+  'code': colorSwatch('code', 'color')
+};
+
+const accentScale = getColorScale('accent-color');
+const informativeScale = getColorScale('informative-color');
+const negativeScale = getColorScale('negative-color');
+const noticeScale = getColorScale('notice-color');
+const positiveScale = getColorScale('positive-color');
+const grayScale = getColorScale('gray');
+const blueScale = getColorScale('blue');
+const redScale = getColorScale('red');
+const orangeScale = getColorScale('orange');
+const yellowScale = getColorScale('yellow');
+const chartreuseScale = getColorScale('chartreuse');
+const celeryScale = getColorScale('celery');
+const greenScale = getColorScale('green');
+const seafoamScale = getColorScale('seafoam');
+const cyanScale = getColorScale('cyan');
+const indigoScale = getColorScale('indigo');
+const purpleScale = getColorScale('purple');
+const fuchsiaScale = getColorScale('fuchsia');
+const magentaScale = getColorScale('magenta');
+const pinkScale = getColorScale('pink');
+const turquoiseScale = getColorScale('turquoise');
+const brownScale = getColorScale('brown');
+const silverScale = getColorScale('silver');
+const cinnamonScale = getColorScale('cinnamon');
+
+const scaleSwatches = {
+  ...Object.fromEntries(accentScale),
+  ...Object.fromEntries(informativeScale),
+  ...Object.fromEntries(negativeScale),
+  ...Object.fromEntries(noticeScale),
+  ...Object.fromEntries(positiveScale),
+  ...Object.fromEntries(grayScale),
+  ...Object.fromEntries(blueScale),
+  ...Object.fromEntries(redScale),
+  ...Object.fromEntries(orangeScale),
+  ...Object.fromEntries(yellowScale),
+  ...Object.fromEntries(chartreuseScale),
+  ...Object.fromEntries(celeryScale),
+  ...Object.fromEntries(greenScale),
+  ...Object.fromEntries(seafoamScale),
+  ...Object.fromEntries(cyanScale),
+  ...Object.fromEntries(indigoScale),
+  ...Object.fromEntries(purpleScale),
+  ...Object.fromEntries(fuchsiaScale),
+  ...Object.fromEntries(magentaScale),
+  ...Object.fromEntries(pinkScale),
+  ...Object.fromEntries(turquoiseScale),
+  ...Object.fromEntries(brownScale),
+  ...Object.fromEntries(silverScale),
+  ...Object.fromEntries(cinnamonScale)
+};
+
+
+function CopyInfoMessage() {
+  return (
+    <div className={style({display: 'flex', alignItems: 'center', justifyContent: 'center', gap: 4})}>
+      <InfoCircle styles={iconStyle({size: 'XS'})} />
+      <span className={style({font: 'ui'})}>Press a color to copy its name</span>
```

Comment:
would be nice to also have a link to the style macro reference when that is also merged